v8/ditaa/ditaa.py

```diff
@@ -0,0 +1,95 @@
+"""Custom reST_ directive for ditaa_ integration.
```

Comment:
Could you add a license header, like the ones seen in Nikola core? (Pick any license you like)

Reply:
Sure, added one.